src/test/resources/examples/authentication/controllers/micronaut/Controllers.kt

```diff
@@ -0,0 +1,83 @@
+package examples.authentication.controllers
+
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.QueryValue
+import io.micronaut.security.annotation.Secured
+import io.micronaut.security.authentication.Authentication
+import io.micronaut.security.rule.SecurityRule
```

Comment:
**issue:** This is not the correct import. (`rules` instead of `rule`)

Reply:
thank you